helm/wfl/values.yaml

```diff
@@ -17,16 +17,16 @@ api:
   deployment:
     containerPort: 80
   env:
-    ENVIRONMENT: "wgs-dev"
```

Comment:
Changes to helm charts need a version bump to https://github.com/broadinstitute/wfl/blob/master/helm/wfl/Chart.yaml, otherwise the lint test will fail (failing now)